app/assets/javascripts/pages/invitations.coffee

```diff
@@ -0,0 +1,13 @@
+PubSub.subscribe('assigned.website', (ev, website)->
+  $('#addMember').submit (e) ->
+    e.preventDefault()
+    $.ajax
+      url: Routes.api_v1_invitations_url()
+      type: 'post'
+      dataType: 'json'
+      data: { member: { email: $('#addMember').find('#getEmail').val(), website: EpicLogger.pickWebsite() } }
```

Comment:
nu prea inteleg de ce folosesti `pickWebsite()` pentru ca nu se potriveste. Mai ales ca ai website sus in argumente dat de catre pubsub. Daca vrei sa folosesti o functie din EpicLogger atunci ar fi mai ok sa faci alta `getWebsite` care sa iti returneze websitu